packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -128,7 +129,9 @@ const Redirect = ({
                 // though can't use "verification_code" as name param
                 // as there is general logic within client-store
                 // which removes anything which resembles code=XYZ
-                history.push(`${routes.wallets_withdrawal}?verification=${verification_code?.payment_withdraw}`);
+                history.push(
+                    `${routes.wallets_withdrawal}?verification=${verification_code?.payment_withdraw}&walletid=${loginid}`
+                );
```

Comment:
Appending the loginid as `walletid` in the URL as any param with `loginid` will be cleared by client-store after redirection.